addons/web/static/src/scss/form_view.scss

```diff
@@ -191,6 +191,10 @@
                 margin: $o-statusbar-buttons-vmargin 3px $o-statusbar-buttons-vmargin 0;
                 padding-top: 2px;
                 padding-bottom: 2px;
+
+                > .o_button_icon {
+                    margin-right: $o-statbutton-spacing;
```

Comment:
could be 4px. meh'.. this way we are aligned with spacing with other button icons. but this variable is not really related with statusbar_buttons.. :shrug: